includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -2453,6 +2459,50 @@ private function remove_disallowed_descendants( DOMElement $node, $allowed_desce
 		}
 	}
 
+	/**
+	 * Loop through node's siblings and remove them.
+	 *
+	 * @param DOMElement $node      Node.
+	 * @param string     $spec_name Spec name.
+	 */
+	private function remove_disallowed_siblings( DOMElement $node, $spec_name ) {
+		if ( ! $node->previousSibling || ! $node->nextSibling ) {
```

Comment:
This condition doesn't seem quite right. It should only bail if the node lacks _both_ the previous and next, not whether it lacks _either_, right?

```suggestion
		if ( ! $node->previousSibling && ! $node->nextSibling ) {
```